src/_includes/navbar.html

```diff
@@ -1,8 +1,7 @@
 <header class="navbar">
   <ul aria-label="Main navigation">
     <li>
-      <a href="/"
-        ><img src="/assets/images/d-marque.png" role="presentation" alt="" /> Home</a
+      <a href="/" aria-label="dxw Accessibility Manual Homepage"><img src="/assets/images/d-marque.png" role="presentation" alt="" /> dxw Accessibility Manual</a
```

Comment:
Thanks @calumryan . Do we need an alt text value here?

Reply:
No, the empty alt here indicates it's a decorative image